### PR TITLE
Shared ranking

### PR DESCRIPTION
**Pull request by mjoyce91**
- Ranking is shared across users
- Fix an issue where if a user lost their organization permissions, their post_user role in TalentMAP would persist

### CODE REVIEW

File: talentmap_api/fsbid/services/employee.py [hunk 1]
```diff
@@ -51,20 +51,19 @@ def map_group_to_fsbid_role(jwt_token):
     Updates a user roles based on what we get back from FSBid
     '''
     roles = jwt.decode(jwt_token, verify=False).get('role')
-    print(roles)
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/employee.py:54]: 😢

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🥮 🍬 👍

**Comment by elizabeth-jimenez** [comment]
only thing to keep in mind is that the `user_id` for all rankings now belongs to the last person to edit the Short List. Like even if Wendy ranks 1-8 and then Preston comes in and switches 7 with 8, the `user_id`s for 1-6 (in addition to 7 and 8) now also belong to the Preston

**Comment by mjoyce91** [comment]
> only thing to keep in mind is that the `user_id` for all rankings now belongs to the last person to edit the Short List. Like even if Wendy ranks 1-8 and then Preston comes in and switches 7 with 8, the `user_id`s for 1-6 (in addition to 7 and 8) now also belong to the Preston

good point, I think that's fine unless we get to a place where we want to display the "last editing user"

**Comment by elizabeth-jimenez** [comment]
> > only thing to keep in mind is that the `user_id` for all rankings now belongs to the last person to edit the Short List. Like even if Wendy ranks 1-8 and then Preston comes in and switches 7 with 8, the `user_id`s for 1-6 (in addition to 7 and 8) now also belong to the Preston
> 
> good point, I think that's fine unless we get to a place where we want to display the "last editing user"

💯  agreed

**Pull request merged**